exosphere-shared/example-apps/docker-service/application.yml

```diff
@@ -0,0 +1,11 @@
+name: docker-app
+description: An application that relies on dockerhub images
+version: 0.0.1
+
+services:
+  public:
+    local-service:
+      docker_image: exospheredev/local-service
+      location: ./local-service
```

Comment:
I don't think this field can be used to deploy a service to Dockerhub right? Because it's `exo-setup` that generates the Docker images, and that uses a certain naming convention. The user can name the image whatever he/she wants in this `docker_image` field, but that won't be the name of the image that was built in `exo setup` (or the one that will be deployed in `exo deploy`).

I think it makes sense to allow for any configuration, and we can do that by having one of  either `docker_image` or `location`. That would still allow the user to use either a remote docker image or their local service: you list the `location` field, you use a local service; you list the `docker_image` field, you use a remote service. But when both `docker_image` and `location` are included, the `docker_image` field will always be ignored it seems like. Does that make sense? What do you think?

Or maybe it's a bit confusing that we call one of the fields `location` when a service's location can be either local or remote..